puppet.py

```diff
@@ -27,6 +27,22 @@ def check_disabled():
         sudo('test -f {0} && echo DISABLED || echo ENABLED'.format(lockfile))
 
 @task
+def regenerate_certificate():
+    """Regenerate a node certificate"""
+    if len(env.hosts) > 1:
+      abort("This task should only be run on one host at a time.")
```

Comment:
It would be good to introduce a newline after the `abort()` invocation to make the code easier to read.